kv-tokens/primitives.json

```diff
@@ -5,33 +5,35 @@
         "greenBlue": "#00C5C5",
         "red": "#E04848",
         "redDark": "#BA342C",
-        "gray0": "#212121",
-        "gray1": "#757575",
-        "gray2": "#C4C4C4",
-        "gray3": "#F5F5F5",
-        "gray4": "#fafafa",
-        "green": {
-            "110": "#26985D",
-            "105": "#28A062",
+        "gray": {
+            "800": "#212121",
+            "500": "#757575",
+            "300": "#C4C4C4",
+            "100": "#F5F5F5",
+            "50": "#fafafa"
+        },
```

Comment:
Just wanted to clarify, will these eventually become `gray-800` or `brand-550` in actual usage?

Reply:
@mcstover correct